docker-compose.yaml

```diff
@@ -41,7 +43,7 @@ services:
       - appsmith:/appsmith-stacks
   
   controller:
-    image: backend-v1
+    image: simpipe-backend-v4
```

Comment:
You should probably use tags and not the image name to have multiple versions.

Somethink like:

  - `simpipe-backend:v1` and `simpipe-backend:v4`